cmake/Modules/FindJson.cmake

```diff
@@ -15,7 +15,7 @@ set(Json_FIND_QUIETLY TRUE)
 libfind_pkg_detect(Json nlohmann_json FIND_PATH nlohmann/json.hpp)
 libfind_process(Json)
 
-if (Json_FOUND)
+if (Json_FOUND AND Json_PKGCONF_VERSION VERSION_GREATER_EQUAL 3.6.0)
```

Comment:
can probably be removed if using count() instead of contains()

Reply:
I can remove it but this was intentional. We still need at least 3.x because of exceptions among other things. 

That's why I thought to just go with contains, because I doubt any disto that has 3.x will have a version older than 3.6.0.

If you're still not convinced, I'll lower the check to whatever version it is that exceptions were actually introduced in, but removing it will lead to a failed build in 18.04 because it will find an incompatible version in system headers.

Reply:
1804 support will drop when they anounce EOL which is in a couple of days. Ubuntu 2004 and 2204 do support 3.6